src/test/spec/v2_runner/test_file.rs

```diff
@@ -23,6 +31,12 @@ pub(crate) struct TestFile {
     #[allow(unused)]
     pub(crate) bucket_name: Option<String>,
     pub(crate) data: Option<TestData>,
+    #[cfg(feature = "csfle")]
```

Comment:
I would have liked to use `#[serde(flatten)]` with a struct defined elsewhere for clarity, but that continues to be incompatible with `deny_unknown_fields` :(